module/VuFind/src/VuFind/View/Helper/Bootstrap3/Search.php

```diff
@@ -38,7 +38,7 @@
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     https://vufind.org/wiki/development Wiki
  */
-class Search extends \VuFind\View\Helper\AbstractSearch
+class Search extends \VuFind\View\Helper\Bootstrap5\Search
```

Comment:
Did you forget to remove the code from this class? It looks to me like the contents are identical to the Bootstrap5 version, but maybe I'm missing something.

Reply:
And I thought I even double-checked what I did... but yes, I forgot. Now fixed.